docs/tutorials/quickstart-guide.md

```diff
@@ -100,7 +100,8 @@ To deploy your app onto Kubernetes,
 In this example, we deploy the `stage` environment:
 
 ```sh
-kubectl apply -f k8s/stage     # deploys your app with stage settings onto the default namespace
+# deploys your app with stage settings onto the default namespace
+kubectl apply -f k8s/stage
```

Comment:
minor: could prefix with a `$` as we do in other place.

Reply:
Oh, nuts, I _knew_ I'd miss one somewhere 🤦  - fixed, nice catch